dist/ember-charts.js

```diff
@@ -3036,7 +3117,13 @@ Ember.Charts.TimeSeriesComponent = Ember.Charts.ChartComponent.extend(Ember.Char
   }).property('_groupedLineData.@each.values'),
   xBetweenSeriesDomain: Ember.computed.alias('lineSeriesNames'),
   xWithinSeriesDomain: Ember.computed.alias('lineDataExtent'),
-  maxNumberOfLabels: Ember.computed.alias('numXTicks'),
+  maxNumberOfLabels: Ember.computed(function(key, newValue) {
+    if (this.get('dynamicXAxis')) {
+      return newValue != null ? newValue : 10;
```

Comment:
Where did the 10 come from? Abstract this into a constant such as `DEFAULT_NUM_X_TICKS` to avoid "magic numbers"